File: app/src/main/java/me/ykrank/s1next/data/api/app/model/AppLoginResult.java
package me.ykrank.s1next.data.api.app.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class AppLoginResult {
    private String uid;
    @JsonProperty("sid")
    private String secureToken;
    @JsonProperty("username")
    private String name;

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getSecureToken() {
        return secureToken;
    }

    public void setSecureToken(String secureToken) {
        this.secureToken = secureToken;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/me/ykrank/s1next/data/api/app/model/AppLoginResult.kt
package me.ykrank.s1next.data.api.app.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class AppLoginResult {
    var uid: String? = null
    @JsonProperty("sid")
    var secureToken: String? = null
    @JsonProperty("username")
    var name: String? = null
}